_posts/es/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,262 @@
+sus pares.  Si más tarde un peer necesita reconectarse, quizás después de perder datos,
+puede solicitar el archivo de copia de seguridad.  El par puede utilizar una clave derivada
+de su cartera para descifrar el archivo y utilizar su contenido para recuperar
+el último estado de todos sus canales.  Esto puede considerarse una
+forma mejorada de [copias de seguridad estáticas de canales][topic static channel
+copias de seguridad].  El PR fusionado añade soporte para crear, almacenar y recuperar
+las copias de seguridad encriptadas.  Como se indica en los mensajes de confirmación, la función
+aún no ha sido completamente especificada o adoptada por otras implementaciones de LN.
+
+- [Core Lightning #5670][] y [#5956][core lightning #5956] realizan
+varias actualizaciones de su implementación de [doble financiación][tema doble
+financiación] basándose tanto en los cambios recientes de la [especificación][tornillos
+#851] y los comentarios de los evaluadores de interoperabilidad.  Además, un
+RPC `upgradewallet` para mover todos los fondos en salidas envueltas en P2SH
+a salidas segwit nativas, lo que es necesario para abrir canales interactivos.
+
+- [Core Lightning #5697][] añade un RPC `signinvoice` que firmará una
+[BOLT11][] factura.  Anteriormente, CLN sólo firmaba una factura cuando
+tenía la imagen previa para el hash [HTLC][tema HTLC], asegurando que sería
+posible reclamar un pago a la factura.  Este RPC puede anular
+que el comportamiento, que podría (por ejemplo) ser utilizado para enviar una factura
+ahora y más tarde utilizar un plugin para recuperar la preimagen de otro
+programa.  Cualquiera que utilice este RPC debe ser consciente de que cualquier tercero
+que tiene conocimiento anterior del preimage para un pago previsto para
+su nodo puede reclamar ese pago antes de que llegue. Eso no sólo
+roba su dinero pero, porque usted firmó la factura, genera varias
+pruebas muy convincentes de que se le pagó (estas pruebas son tan convincentes
+que muchos desarrolladores de LN la llaman *prueba de pago*).
+
+- [Core Lightning #5960][] añade una [política de seguridad][cln security.md]
+que incluye direcciones de contacto y claves PGP.
+
+- [LND #7171][] actualiza el RPC `signrpc` <!--sic--> para que sea compatible con el
+último [borrador BIP][borrador bip musig] para [MuSig2][tema musig].  El RPC ahora crea
+sesiones vinculadas a un número de versión del protocolo MuSig2 para que todas las
+operaciones dentro de una sesión utilicen el protocolo correcto. Un
+problema de seguridad con una versión antigua del protocolo MuSig2 fue
+mencionado en [Newsletter #222][news222 musig2].
+
+- [LDK #2002][] añade soporte para reenviar automáticamente [pagos espontáneos
+pagos][tema pagos] que no tienen éxito inicialmente.
+
+- [BTCPay Server #4600][] actualiza la [selección de moneda][tema selección de moneda] para su [payjoin][tema payjoin]
+para tratar de evitar la creación de transacciones con *entradas 
```

Comment:
```suggestion
para tratar de evitar la creación de transacciones con *entradas
```

Reply:
If you look in the box below this comment, it tells you that “6 checks are failing”. When you click on “Details” and then link on “the logs” in “Please check the logs.” under Deploy failed, it does tell you which rules caused the failure.